### PR TITLE
net-p2p/syncthing: bump to 0.13.9

### PR DESCRIPTION
**Pull request by sbraz**
@djc just small fixes and an enhancement https://github.com/syncthing/syncthing/releases/tag/v0.13.8
I removed the remaining 0.12.x version.

### CODE REVIEW

**Comment by sbraz** [comment]
Looks like there's a new release. Please don't merge this now.

**Comment by gktrk** [comment]
@gentoo/proxy-maint

**Comment by sbraz** [comment]
Fixed, it's good to go now

File: net-p2p/syncthing/syncthing-0.13.9.ebuild [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+	if use tools ; then
+		# separate user for relaysrv
+		enewgroup ${PN}-relaysrv
+		enewuser ${PN}-relaysrv -1 -1 /var/lib/${PN}-relaysrv ${PN}-relaysrv
+	fi
+}
+
+src_prepare() {
+	epatch "${FILESDIR}/relaysrv.systemd.patch"
```
Comment (gktrk) [inline net-p2p/syncthing/syncthing-0.13.9.ebuild]: should probably switch to eapply or just use PATCHES here and get rid of src_prepare

**Comment by gktrk** [comment]
Merged https://github.com/gentoo/gentoo/commit/49416dec7eb32a66937f1931dae662a865be25ff. Thanks!

**Pull request closed without merging**